src/View/ColumnView.vala

```diff
@@ -56,21 +56,20 @@ namespace FM {
             return false;
         }
 
-        protected override Marlin.ZoomLevel get_set_up_zoom_level () {
-            var zoom = Marlin.column_view_settings.get_enum ("zoom-level");
-            Marlin.column_view_settings.bind ("zoom-level", this, "zoom-level", GLib.SettingsBindFlags.SET);
+        protected override void set_up_zoom_level () {
+            Marlin.column_view_settings.bind ("zoom-level",
+                                                           this, "zoom-level",
+                                                           GLib.SettingsBindFlags.DEFAULT);
```

Comment:
I don't really think this is easier to read than having it inline, but if it must be shortened then code style is for it to be done like:
```suggestion
            Marlin.column_view_settings.bind (
                "zoom-level",
                this, "zoom-level",
                GLib.SettingsBindFlags.DEFAULT
            );
```